comment/templatetags/comment_tags.py

```diff
@@ -139,3 +140,48 @@ def include_bootstrap():
 def render_field(field, **kwargs):
     field.field.widget.attrs.update(kwargs)
     return field
+
+
+@register.filter(name='add_one_arg')
+def add_one_arg(arg1, arg2):
+    """
+    Use this filter to when you need to pass more than 2 arguments to your filter.
+
+    Args:
+        arg1 (`Any`): First Argument
+        arg2 (`Any`): Second Argument
+
+    Returns:
+        tuple: a tuple containing both the arguments in the order that they were passed.
+    """
+    return arg1, arg2
+
+
+@register.filter(name='has_reacted')
+def has_reacted(comment_and_user, reaction):
```

Comment:
Sorry for the late review, I saw that you have written the test already but it is really worthy to refactor the code here.
Instead of using these two filter, we can use a simple tag directly as follows:
```python
@register.simple_tag(name='has_reacted')
def has_reacted(comment, user, reaction):
    if user.is_authenticated:
        reaction_type = getattr(ReactionInstance.ReactionType, reaction.upper(), None)
        if not reaction_type:
            raise template.TemplateSyntaxError(
                'Reaction must be an valid ReactionManager.RelationType. {} is not'.format(reaction)
                )
        return ReactionInstance.objects.filter(
            user=user,
            reaction_type=reaction_type.value,
            reaction__comment=comment
            ).exists()

    return False

```

Use in temple:
```jinja2
{% if hast_reacted comment=obj user=user reaction="like" %}"#ffc966"{% else %}"none"{% endif %}
```

or if you like you can return the color directly instead of the boolean value.

Reply:
Okay, I will do that. Returning the color from the template tag function would make it a little too complex for the users to change the icon color if they want. I will keep the color inside the template.